packages/playground/src/exporter.ts

```diff
@@ -269,7 +269,9 @@ ${codify(stringifiedCompilerOptions, "json")}
     const ts = sandbox.getText()
     const preview = ts.length > 200 ? ts.substring(0, 200) + "..." : ts.substring(0, 200)
 
-    const code = "```\n" + preview + "\n```\n"
+    const jsx = getJsxEmitText(sandbox.getCompilerOptions().jsx)
```

Comment:
I reached for `sandbox.getCompilerOptions().jsx` first but that seemed to `undefined` regardless of the tsconfig settings. Easy to make this mistake so idealy `sandbox.getCompilerOptions().jsx` would "just work".

Reply:
Interesting, from a cursory place I don't see how this code would get out of sync with the compiler's JSX settings, will keep my eye out - thanks